packages/e2e/features/ui/components/authenticator/sign-in-with-phone.feature

```diff
@@ -38,3 +37,11 @@ Feature: Sign In with Phone Number
     And I type the valid password "VALID_PASSWORD"
     And I click the "Sign In" button
     Then I see "Change Password"
+
+  @next @react @vue @angular
+    Scenario: Phone number field autocompletes phone number
+      And "Phone Number" field autocompletes "username"
```

Comment:
Did you see some weird behavior that required the triple-indentation?

```suggestion
  @next @react @vue @angular
  Scenario: Phone number field autocompletes phone number
    And "Phone Number" field autocompletes "username"
```

Reply:
This was a common formatting error in our feature files. I've mostly seen tags on the same indentation as `Scenario` keywords. I wonder if there's a plugin we can use to lint Gherkin syntax.

Reply:
No, this is probably on me. Adjusting now.

Reply:
@slaymance Man, that would be awesome! I'll take a look too.